src/kitovu/cli.py

```diff
@@ -62,3 +66,51 @@ def validate(config: typing.Optional[pathlib.Path] = None) -> None:
         syncing.validate_config(config, CliReporter())
     except utils.UsageError as ex:
         raise click.ClickException(str(ex))
+
+
+@cli.command()
+def docs() -> None:
+    """Open the documentation in the browser."""
+    # FIXME: make version aware
+    webbrowser.open_new_tab('https://kitovu.readthedocs.io/en/latest')
+
+
+DEFAULT_EDITORS = [
+    'vim',
+    'emacs',
+    'nano',
+    'editor',
+    'notepad',
+]
+
+
+@cli.command()
+@click.option('--config', type=pathlib.Path, help="The configuration file to edit")
+@click.option('--editor', type=str, help=f"The command of the editor to use. \
+              Default: $EDITOR or the first existing out of {', '.join(DEFAULT_EDITORS)}")
+def edit(config: typing.Optional[pathlib.Path] = None, editor: typing.Optional[str] = None) -> None:
+    """Edit the specified configuration file."""
+    if editor is None and 'EDITOR' in os.environ:
+        editor = os.environ['EDITOR']
+    editor_path: typing.Optional[str] = _get_editor_path(editor)
+
+    if editor_path is None:
+        if editor is None:
+            print('Could not find a valid editor')
+        else:
+            print(f"Could not find the editor {editor}")
+        sys.exit(1)
```

Comment:
Raise a [ClickException](http://click.pocoo.org/5/api/#click.ClickException) instead? Maybe even from `_get_editor_path` (instead of returning `None`) instead of here.